cmd/tf-operator.v1/app/server.go

```diff
@@ -99,8 +99,8 @@ func Run(opt *options.ServerOption) error {
 	if err != nil {
 		log.Fatalf("Error building kubeconfig: %s", err.Error())
 	}
-	kcfg.QPS = 10
-	kcfg.Burst = 20
+	kcfg.QPS = 50
+	kcfg.Burst = 100
```

Comment:
Why do we need to increase default QPS and Burst?

Reply:
Done